node/actors/pb/agents.proto

```diff
@@ -17,39 +17,80 @@ message Ping {
 
 // Sent by the control API actor to an agent actor to request a workload be started
 message StartWorkload {
+  repeated string argv = 1;
+  string description = 2;
+  string environment = 3;
+  bool essential = 4;
+  string hash = 5;
+  HostServicesConfig host_service_config = 6;
+  string jsdomain = 7;
+  string retried_at = 8;
+  int32 retry_count = 9;
+  string sender_public_key = 10;
```

Comment:
we don't need the sender key, or the target node, etc. Keep in mind that the protos are for internal use and so we can remove any fields we no longer need after the API server has validated them.

Reply:
yea, I was just copying a 1:1 over and was going to audit after the fact, but ill pull these two out